pkgs/development/python-modules/mizani/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, matplotlib
+, palettable
+, pandas
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mizani";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "has2k1";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04r53dp5jbklv8l9ncgc5wiq0gx25y73h65gmmbbfkxwgsl3w78l";
+  };
+
+  postPatch = ''
+    substituteInPlace pytest.ini --replace " --cov=mizani --cov-report=xml" ""
+  '';
+
+  propagatedBuildInputs = [ matplotlib palettable pandas ];
+
+  checkInputs = [ pytestCheckHook ];
```

Comment:
Missing pythonImportsCheck.

Reply:
Good catch! Done in https://github.com/NixOS/nixpkgs/pull/137458/commits/0bf323812d46ff811fcb3e7f74fdddf5d64c8497